p2p/transport/quic/listener.go

```diff
@@ -84,7 +82,7 @@ func (l *listener) Accept() (tpt.CapableConn, error) {
 	}
 }
 
-func (l *listener) setupConn(qconn quic.Connection) (*conn, error) {
+func (l *listener) setupConn(qconn quic.Connection) (_c *conn, _err error) {
```

Comment:
Is the `_` needed? I've never seen this pattern.

Reply:
not needed, I don't want to shadow or deal with shadowed vars. I also don't want this to be used in the body of the function.

I also want to avoid the use of the named return values _except_ for this deferred error handling.

If you have a better naming suggestion I'm open to it.